workflows/validate.libsonnet

```diff
@@ -35,44 +35,77 @@ local validationJob = _validationJob(false);
     + step.withIf('${{ !fromJSON(env.SKIP_VALIDATION) }}')
     + step.withRun(common.makeTarget(target)),
 
-  test: setupValidationDeps(
-    validationJob
-    + job.withSteps([
-      validationMakeStep('test', 'test'),
-    ])
-  ),
+  // Test jobs
+  collectPackages: job.new()
+                   + job.withSteps([
+                     common.checkout,
+                     common.fixDubiousOwnership,
+                     step.new('gather packages')
+                     + step.withId('gather-tests')
+                     + step.withRun(|||
+                       echo "packages=$(find . -path '*_test.go' -printf '%h\n' \
+                         | grep -e "pkg/push" -e "integration" -e "operator" -e "lambda-promtail" -v \
```

Comment:
so to be clear, we're not running the integration or operator tests anymore? and pkg/push and lambda-promtail will be run as separate jobs?

Reply:
not running operator, correct, it has it's own workflow that's guarded on changes to that folder. we are running integration tests just in a different, parallel job